App/Containers/ThreadInvite.js

```diff
@@ -26,6 +26,7 @@ class ThreadInvite extends React.PureComponent {
   }
 
   cancel = () => {
+    // TODO: remove thread takes thread name... what is key?
     this.props.removeThreadRequest(this.state.key)
```

Comment:
@andrewxhill this function to remove the thread needs to take the thread name (until we fix that API)... wasn't sure what `this.state.key` represents, but let's make sure we're passing thread name here?

Reply:
i think we'll have a fix for it later today from the go side, so maybe just let it be until framework update

Reply:
cool